test/factories.py

```diff
@@ -160,6 +160,7 @@ class Meta:
 class ScopeFactory(ODPModelFactory):
     class Meta:
         model = Scope
+        sqlalchemy_get_or_create = ('id', 'type')  # Force the factory to fetch if keys match
```

Comment:
This can be removed - we can rather use the FactorySession to get the relevant scope object in the test function. Note that in `test/api/conftest.py` there is an autouse fixture which creates all the system scopes for every test, since scopes are needed just about everywhere.